src/test/kotlin/no/nav/syfo/personstatus/api/v2/DialogmotekandidatPersonoversiktStatusApiV2Spek.kt

```diff
@@ -0,0 +1,168 @@
+                audience = externalMockEnvironment.environment.azure.appClientId,
+                issuer = externalMockEnvironment.wellKnownVeilederV2.issuer,
+                navIdent = UserConstants.VEILEDER_ID,
+            )
+
+            val personIdentDefault = PersonIdent(ARBEIDSTAKER_FNR)
+            val mockKafkaConsumerOppfolgingstilfellePerson =
+                TestKafkaModule.kafkaConsumerOppfolgingstilfellePerson
+            val oppfolgingstilfellePersonTopicPartition = oppfolgingstilfellePersonTopicPartition()
+            val kafkaOppfolgingstilfellePersonRelevant = generateKafkaOppfolgingstilfellePerson(
+                personIdent = personIdentDefault,
+                virksomhetsnummerList = listOf(
+                    UserConstants.VIRKSOMHETSNUMMER_DEFAULT,
+                    Virksomhetsnummer(UserConstants.VIRKSOMHETSNUMMER_2),
+                )
+            )
+            val kafkaOppfolgingstilfellePersonRecordRelevant = oppfolgingstilfellePersonConsumerRecord(
+                kafkaOppfolgingstilfellePerson = kafkaOppfolgingstilfellePersonRelevant,
+            )
+            val kafkaOppfolgingstilfellePersonService = TestKafkaModule.kafkaOppfolgingstilfellePersonService
+
+            beforeEachTest {
+                database.connection.dropData()
+
+                clearMocks(mockKafkaConsumerOppfolgingstilfellePerson)
+                every { mockKafkaConsumerOppfolgingstilfellePerson.commitSync() } returns Unit
+                every { mockKafkaConsumerOppfolgingstilfellePerson.poll(any<Duration>()) } returns ConsumerRecords(
+                    mapOf(
+                        oppfolgingstilfellePersonTopicPartition to listOf(
+                            kafkaOppfolgingstilfellePersonRecordRelevant,
+                        )
+                    )
+                )
+            }
+
+            describe("Hent dm2-kandidater for enhet") {
+                val url = "$personOversiktApiV2Path/enhet/${UserConstants.NAV_ENHET}"
+
+                it("should return NoContent, if there is a person with a tilfelle, who is kandidat, but has an open DM2 invitation") {
+                    kafkaOppfolgingstilfellePersonService.pollAndProcessRecords(
+                        kafkaConsumer = mockKafkaConsumerOppfolgingstilfellePerson,
+                    )
+                    setAsKandidat(database)
+                    setDialogmotestatus(database, DialogmoteStatusendringType.INNKALT)
+                    setTildeltEnhet(database)
+
+                    with(
+                        handleRequest(HttpMethod.Get, url) {
+                            addHeader(HttpHeaders.Authorization, bearerHeader(validToken))
+                        }
+                    ) {
+                        response.status() shouldBeEqualTo HttpStatusCode.NoContent
+                    }
+                }
+
+                it("should return person who is kandidat if he/she has tilfelle, is kandidat, and a cancelled dm2") {
```

Comment:
Her kan man bruke `singular they` i stedet for "he/she" hvis man ønsker 👍

Reply:
supa-woke!